### PR TITLE
Fix deployment

### PR DESCRIPTION
**Pull request by Naimaoye**

### CODE REVIEW

**Comment by Naimaoye** [comment]
#### What does this PR do?
set up API endpoints on heroku
#### Description of Task to be completed?
Testing API endpoints thoroughly
#### How should this be manually tested?
-cd into the quickcredit-App folder
-type npm run start-dev
-test heroku URL on postmann
#### Any background context you want to provide?
-No
#### Screenshots (if appropriate)
isAdmin-false


#### Questions:
-No questions yet

File: server/models/userss.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    {
+        id: 3,
+        email: 'adebayo@quickcredit.com',
+        password: Authenticate.hashPassword('password'),
+        firstName: 'john',
+        lastName: 'doe',
+        address: 'Lekki phase I, Lagos',
+        status: 'verified',
+        isAdmin: true,
+      //  token: Auth.generateToken({'adebayo@quickcredit.com', true})
```
Comment (efalayi) [inline server/models/userss.js]: Please remove all instances of dead code.

File: server/controller/loans_controller.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+      interest: userLoan.interest,
+    };
+    return res.status(200).send({
+      status: 200,
+      data: updatedData,
+    });
+  }
+}
+
+export default Loancontroller;
```
Comment (efalayi) [inline server/controller/loans_controller.js]: Please fix this new line error.

File: server/models/userss.js (hunk 1 above)
Comment (Naimaoye) [reply server/models/userss.js]: #### What does this PR do?
fixes all dead codes
#### Description of Task to be completed?
upload to heroku
#### How should this be manually tested?
-cd into the quickcredit-App folder
-type npm run start-dev
-test heroku URL on postmann
#### Any background context you want to provide?

#### Screenshots (if appropriate)
-No screenshot
#### Questions:
-No question

**Review by efalayi — APPROVED** [review]
LGTM. You can merge.

**Pull request merged**